blueprints/going-to-production-with-healthcare-apps.html.md

```diff
@@ -176,4 +176,4 @@ Finally run `fly deploy --path fly.production.yml` to deploy your application, t
 
 ## Wrap-up
 
-Fly.io is a secure, productive platform for deploying HIPAA-compliant healthcare applications. The platform provides the necessary security controls and features to ensure that PHI is protected in accordance with HIPAA requirements while giving development teams the flexibility to ship often and scale quickly.
+Fly.io is a secure, productive platform for deploying HIPAA-compliant healthcare applications. It provides the necessary security controls and features to ensure that PHI is protected in accordance with HIPAA requirements while giving development teams the flexibility to ship often and scale quickly.
```

Comment:
leave as-is. it's more clear when the subject of the sentence is named

Reply:
I have made the required change. Thank you for the review.